fission/main.go

```diff
@@ -61,12 +61,16 @@ func main() {
 	fnHeaderFlag := cli.StringSliceFlag{Name: "header, H", Usage: "request headers"}
 	fnEntryPointFlag := cli.StringFlag{Name: "entrypoint", Usage: "entry point for environment v2 to load with"}
 	fnBuildCmdFlag := cli.StringFlag{Name: "buildcmd", Usage: "build command for builder to run with"}
+	fnSecretFlag := cli.StringFlag{Name: "secret", Usage: "function access to secret"}
+	fnSecretnsFlag := cli.StringFlag{Name: "secretns", Usage: "namespace of secret"}
```

Comment:
I was thinking that we may not need to provide the user an option for secret namespace and configMap namespace. Mainly because we dont provide the user an option to specify namespace for their function.
If the reason that we want to give this as an option is that the secret names and configmap names shouldnt collide, then we can probably prefix or suffix them with function names since they are unique. This is just a thought.